### DISCUSSION TITLE
Larger/smaller while holding

### DISCUSSION THREAD

**Discussion by onemore-creator** [rxhanson/Rectangle#1413] [Ideas]
I find it annoying that I should always click if I want to make the window larger/smaller. I'd like to introduce scaling while holding the corresponding keys. I can do this on my own, so I mostly want to confirm that this feature could be useful for somebody else

**Comment by rxhanson** [comment]
This is commonly asked for, so it would be a welcome change. You might have to modify the shortcut library (a fork that I maintain). I’m away from my home setup at the moment, but can answer questions if you need help.

**Reply by onemore-creator** [reply]
Sure, thanks! I am glad to hear that it is required. I will investigate the code first then
